### DISCUSSION TITLE
Facebook Auth

### DISCUSSION THREAD

**Discussion by dkornilove** [umputun/remark42#1113] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/55177640/130463077-dd761aca-9178-47cb-be59-c00e630df831.png)
Am I right saying the following?
Marked as 1 is AUTH_FACEBOOK_CID
Marked as 2 is AUTH_FACEBOOK_CSEC

Getting this error when trying to auth
![image](https://user-images.githubusercontent.com/55177640/130463370-a52892a1-07f2-499b-a176-e56864f4fc12.png)

Other providers working well

**Comment by dkornilove** [comment]
![image](https://user-images.githubusercontent.com/55177640/130464203-c66af5e9-0907-4640-8fc3-8ad042ae1d79.png)

My facebook login settings

**Comment by dkornilove** [comment, marked as answer]
Sorry, my bad
AUTH_FACEBOOK_CSEC must be taken from here
![image](https://user-images.githubusercontent.com/55177640/130478716-b63c85ec-abae-4eba-9203-964a1636810f.png)

**Reply by paskal** [reply]
@dkornilove in order to help you we need to see everything Facebook- and auth-related from the docker-compose log: there should be errors related to the lack of Facebook provider setup. Another possibility is that that functionality was added just recently, to validate if it's the case please run not `:latest` but `:master` image and see if it will appear.
